components/txn_types/src/lock.rs

```diff
@@ -188,6 +210,11 @@ impl Lock {
                 b.encode_u64(ts.into_inner()).unwrap();
```

Comment:
I'm just curious why line 180
```
b.encode_var_u64(self.ts.into_inner()).unwrap();
```
uses `encode_var_u64`?

Reply:
A real timestamp large enough so var int doesn't save space. Using var int only slows down encoding and decoding.

I guess there wasn't a reason. The original author just didn't think much about it.